pep-9999.rst

```diff
@@ -260,17 +265,21 @@ The ``notices`` key is an optional key that points to an array of notices that
 the server wishes to communicate to the end user that are not specific to any
 one file.
 
+The ``url`` key represents a unique and absolute URL for the session. This is
```

Comment:
Is this different than the url in the ``Location`` header?

Reply:
I missed that! I don't think so, seems like we can remove this.